examples/bootstrap_hwdefinition.yaml

```diff
@@ -18,10 +18,11 @@
 #############################################################################
 # version the schema in this file so consumers can rationally parse it
 ---
-apiVersion: v1
+apiVersion: 1.0
```

Comment:
using `v1.0` is perfectly fine and the `v` prefix is common with other API's this team would be interacting with. kubernetes, calico...most of them use this prefix.